components/mpas-ocean/src/tracer_groups/Registry_ecosys.xml

```diff
@@ -410,7 +410,7 @@
 				<var name="NH4SurfaceFluxRunoff" array_group="ecosysSurfaceFluxRunoffGRP" units="mmol N m^{-3} m s^{-1}"
 					description="Dissolved Ammonia Surface Flux Due to Runoff"
 				/>
-				<var name="FeSurfaceFluxRunoff" array_group="ecosysSurfaceFluxRunoffSurfaceFluxRunoffGRP" units="mmol Fe m^{-3} m s^{-1}"
+				<var name="FeSurfaceFluxRunoff" array_group="ecosysSurfaceFluxRunoffGRP" units="mmol Fe m^{-3} m s^{-1}"
```

Comment:
I'm sure this is right but probably should be in its own PR.  Do you want to check with Mat Maltrud about that?

Reply:
I told him about it a while ago, but I think he forgot about it